es6-shim.js

```diff
@@ -1498,7 +1498,11 @@
     },
 
     log2: function (value) {
-      return Math.log(value) * Math.LOG2E;
+      if (value === 0) {
+        return -1 / 0;
+      }
+      var e = Math.min(Math.floor(Math.log(value) / Math.LN2), 1023);
```

Comment:
if that's where `1023` comes from, then absolutely a named constant would be ideal.